katcp/client.py

```diff
@@ -504,7 +504,7 @@ def handle_message(self, msg):
         """
         # log messages received so that no one else has to
         if self._logger.isEnabledFor(logging.DEBUG):
-            self._logger.debug(str(msg))
+            self._logger.debug(repr(str(msg)))
```

Comment:
The katcp message class does a different serialisation depending on whether
you do repr or str -- the repr one 'pretty-prints' it, which has the
disadvantage that spaces are unescaped, which means you can no longer
descern from the logs whether you have multiple arguments or a single
argument containing spaces.

On Tue, Sep 8, 2015 at 10:10 AM, Theuns Alberts <notifications@github.com>
wrote:

> In katcp/client.py
> <https://github.com/ska-sa/katcp-python/pull/124#discussion_r38899483>:
>
> > @@ -504,7 +504,7 @@ def handle_message(self, msg):
> >          """
> >          # log messages received so that no one else has to
> >          if self._logger.isEnabledFor(logging.DEBUG):
> > -            self._logger.debug(str(msg))
> > +            self._logger.debug(repr(str(msg)))
>
> repr(str( intended? Why not just repr?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ska-sa/katcp-python/pull/124/files#r38899483>.
>